### ISSUE TITLE
Enhancement: Map Legend / Key

### ISSUE THREAD

**Issue by markongh** [russellporter/openskimap.org#118]
It would be very helpful if a map legend was included somewhere on the website to explain how different features are displayed. I would recommend including the legend in the hamburger button menu to the left of the Search bar.  Some elements I recommend including based on what I see depicted in OpenSkiMap are:

- The Blue and Green dots for the resort (Downhill & Backcountry vs Nordic)
- Colors for Piste difficulty (Easy, Intermediate, Advanced, Expert, Freeride terrain, etc)
- Piste grooming type (if depicted differently - I've seen solid vs dashed lines)
- Lit Pistes (I noticed these have a yellow outline)
- Aerialways

I appreciate this website and use it quite a bit for researching ski locations, and I hope to see it's development continue! Thanks!

**Comment by gr4v1ty69** [comment]
Yes, I support this idea. For example, I have no idea what the exclamation mark means : 

![Image](https://github.com/user-attachments/assets/f9233561-3a8b-464a-9f5a-968fa06dc45d)
